### PR TITLE
Update typescript docs for adding custom commands

### PR DESCRIPTION
**Pull request by kaspatel-mdsol**
This is a PR to update documentation for adding custom commands in typescript that may or may not include external modules.

This is originally sourced from [here](https://github.com/cypress-io/add-cypress-custom-command-in-typescript/issues/2#issuecomment-389870033).

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
@kaspatel-mdsol Thanks for the contribution! Could you please sign our CLA?

File: source/guides/tooling/typescript-support.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ We recommend the following configuration in a {% url "`tsconfig.json`" http://ww
 ```json
 {
   "compilerOptions": {
+    "baseUrl": "./",
```
Comment (bahmutov) [inline source/guides/tooling/typescript-support.md:34]: why do you need to set the base URL?

File: source/guides/tooling/typescript-support.md [hunk 2]
```diff
@@ -65,48 +66,41 @@ For example if you add the command `cy.dataCy` into your {% url "`supportFile`"
 
 ```javascript
 // cypress/support/index.ts
+
+import { MyModule } from "@foo/bar";
```
Comment (bahmutov) [inline source/guides/tooling/typescript-support.md:70]: I really do not like foo / bar example here. What does MyModule do?

**Review by bahmutov — CHANGES_REQUESTED** [review]
I am not sure how this documentation update is any better than the previous doc

**Comment by amirrustam** [comment]
@kaspatel-mdsol feel free to reopen this PR if you can address @bahmutov's feedback.

**Pull request closed without merging**